nomium/shares-logger/src/storage/clickhouse/service.rs

```diff
@@ -6,8 +6,9 @@ use crate::storage::clickhouse::ClickhouseConnectionPool;
 use crate::traits::ShareStorage;
 use async_trait::async_trait;
 use clickhouse::Client;
-use log::{info, error};
+use log::{info, error, warn};
 use std::sync::Arc;
+use rpc_sv2::mini_rpc_client::RpcError;
```

Comment:
Introduced 'RpcError' from 'rpc_sv2' to handle RPC-related errors when storing shares.